app/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "start":
-      "webpack-dev-server --config webpack.dev.js --progress --watch --color",
+      "webpack-dev-server --config webpack.dev.js --watch-poll --progress --watch --color",
```

Comment:
I think you didn't mean to commit this. You can commit part of a file with `git add -p <filename>`